templates/contributing_template.rst

```diff
@@ -0,0 +1,159 @@
+Contribution guide
+==================
+
+Interested in contributing to the Invenio project? There are lots of ways to
+help.
+
+.. rubric:: Code of conduct
+
+Overall, we're **open, considerate, respectful and good to each other**. We
+contribute to this community not because we have to, but because we want to.
+If we remember that, our :ref:`code-of-conduct` will come naturally.
+
+.. rubric:: Get in touch
+
+See :ref:`getting-help` and :ref:`communication-channels`. Don't hesitate
+to get in touch with the Invenio maintainers (listed in ``MAINTAINERS`` file).
+The maintainers can help you kick start your contribution.
+
+Types of contributions
+----------------------
+
+Report bugs
+~~~~~~~~~~~
+- **Found a bug? Want a new feature?** Open a GitHub issue on the applicable
+  repository and get the conversation started (do search if the issue has
+  already been reported). Not sure exactly where, how, or what to do?
+  See :ref:`getting-help`.
+
+- **Found a security issue?** Alert us privately at
+  `info@inveniosoftware.org <info@inveniosoftware.org>`_, this will allow us to
+  distribute a security patch before potential attackers look at the issue.
+
+Translate
+~~~~~~~~~
+- **Missing your favourite language?** Translate Invenio on
+  `Transifex <https://www.transifex.com/inveniosoftware/invenio/>`_
+
+- **Missing context for a text string?** Add context notes to
+  translation strings or report the issue as a bug (see above).
+
+- **Need help getting started?** See our :ref:`translation-guide`.
+
+Write documentation
+~~~~~~~~~~~~~~~~~~~
+- **Found a typo?** You can edit the file and submit a pull request directly on
+  GitHub.
+
+- **Debugged something for hours?** Spare others time by writing up a short
+  troubleshooting piece on
+  https://github.com/inveniosoftware/troubleshooting/.
+
+- **Wished you knew earlier what you know now?** Help write both non-technical
+  and technical topical guides.
+
+Write code
+~~~~~~~~~~
+- **Need help getting started?** See our :ref:`developers-guide`.
+
+- **Need help setting up your editor?** See our
+  :ref:`setting-up-your-environment` guide which helps your automate the
+  tedious tasks.
+
+- **Want to refactor APIs?** Get in touch with the maintainers and get the
+  conversation started.
+
+- **Troubles getting green light on Travis?** Be sure to check our
+  :ref:`style-guide` and the :ref:`setting-up-your-environment`. It will make
```

Comment:
All these ref links won't work. Perhaps better to take out the contribution guide for now.